codecov.yml

```diff
@@ -1 +1,19 @@
-comment: false
+coverage:
+  status:
+    project:
+      default:
+        threshold: 1%
+        branches:
+          - master
+        if_not_found: success
+        if_ci_failed: error
+        # Disable once code base is more mature.
+        informational: true
+        only_pulls: true
+
+comment:
+  layout: "reach, diff, flags, files"
+  behavior: default
+  require_changes: true
+  branches:
+    - "master"
```

Comment:
What does it do?

Reply:
Too many things 😅 

One part is for tweaking the PR comment made by the bot, making sure that it only works for PR targeting master.

The other part is for the Codecov GitHub app that can actually prevent merging PRs when coverage drops significantly. Although I disabled that particular functionality with the `informational: true` flag.